anime/components/AnimeSearch.vue

```diff
@@ -93,50 +216,44 @@
 import { useUserStore } from "~~/stores/userStore";
 import { ref } from "vue";
 import { onMounted } from "vue";
+import { animeRest } from "~~/types/anime";
+import AnimeCard from "./homepage/AnimeCard.vue";
+import RightPageButton from "./RightPageButtonSvg.vue";
+import LeftPageButton from "./LeftPageButtonSvg.vue";
+import AnimeCardLoading from "./homepage/AnimeCardLoading.vue";
 
 const userStore = useUserStore();
 
-const pageExistLeft = ref(false);
-const pageExistRight = ref(true);
-const pageFilteredAnime = ref([] as any);
+const text = ref("");
+const animeResults = ref([] as animeRest[]);
+
+const pageExistLeft = ref<boolean>(false);
+const pageExistRight = ref<boolean>(true);
+const pageFilteredAnime = ref([] as animeRest[]);
+
+const media_genre = ref("");
+const media_year = ref("");
```

Comment:
another thing you could do is make a interface for the anime and set it the store